server/src/com/sixsq/slipstream/ssclj/resources/spec/device_dynamic.cljc

```diff
@@ -26,11 +26,11 @@
 (s/def :cimi.device-dynamic/ethernetThroughputInfo (s/coll-of string?))
 (s/def :cimi.device-dynamic/wifiThroughputInfo (s/coll-of string?))
 (s/def :cimi.device-dynamic/actuatorInfo ::cimi-core/nonblank-string)
-;(s/def :cimi.device-dynamic/status ::cimi-core/nonblank-string)
+(s/def :cimi.device-dynamic/status ::cimi-core/nonblank-string)
```

Comment:
@ssgUPC the status must be a well defined set, like `#{"CONNECTED" "DISCONNECTED" "UNKNOWN"}`, and not a random string

Reply:
@cjdcordeiro the `status` is not a random string, we are basically considering three different string to represents the `status` of an `agent`. Those three different strings are - `Connected`, `Disconnected`, and `Unavailable`. But initially, the `status` value must be set as `Connected`. So that's why we internally decided the `status` should be a string and that is the reason for mentioning the datatype of `status` as a string.

Reply:
so we are in the same page. Please change it to a well defined list of states, instead of string

Reply:
@cjdcordeiro just one question that means, I need to change the `status` into `list` datatype right?? and  I can define it like - `(s/def :cimi.device-dynamic/status (s/coll-of string?))` in that way right?

Reply:
@ssgUPC no. look at https://github.com/mF2C/cimi/blob/master/server/src/com/sixsq/slipstream/ssclj/resources/spec/service.cljc#L10

Reply:
Okay...got it.. Thanks Cris

Reply:
@cjdcordeiro fixed the 'restricted' issue